src/Configuration/test/Kubernetes.ServiceBinding.Test/LegacyConnectorsPostProcessorsTest.cs

```diff
@@ -63,4 +62,38 @@ public void PostgreSqlTest_BindingTypeEnabled()
         Assert.Equal("root.cert", configData["postgres:client:sslrootcert"]);
         Assert.Equal("--cluster=routing-id&opt=val1", configData["postgres:client:options"]);
     }
+
+    [Fact]
+    public void RabbitMQTest_BindingTypeEnabled()
+    {
+        var postProcessor = new RabbitMQLegacyConnectorPostProcessor();
+
+        Dictionary<string, string> configData = GetConfigData(_testBindingName, RabbitMQPostProcessor.BindingTypeKey, Tuple.Create("addresses", "test-addresses"), Tuple.Create("host", "test-host"), Tuple.Create("password", "test-password"), Tuple.Create("port", "test-port"), Tuple.Create("username", "test-username"), Tuple.Create("virtual-host", "test-virtual-host"));
+
+        // BindingType enabled
+        postProcessor.PostProcessConfiguration(GetConfigurationProvider(postProcessor, RabbitMQPostProcessor.BindingTypeKey, true), configData);
+        Assert.Equal("test-addresses", configData["rabbitmq:client:uri"]);
+        Assert.Equal("test-host", configData["rabbitmq:client:server"]);
+        Assert.Equal("test-password", configData["rabbitmq:client:password"]);
+        Assert.Equal("test-port", configData["rabbitmq:client:port"]);
+        Assert.Equal("test-username", configData["rabbitmq:client:username"]);
+        Assert.Equal("test-virtual-host", configData["rabbitmq:client:virtualhost"]);
+    }
+
+    [Fact]
+    public void RedisTest_BindingTypeEnabled()
+    {
+        var postProcessor = new RedisLegacyConnectorPostProcessor();
+
+        Dictionary<string, string> configData = GetConfigData(_testBindingName, RedisPostProcessor.BindingTypeKey, Tuple.Create("host", "test-host"), Tuple.Create("password", "test-password"), Tuple.Create("port", "test-port"), Tuple.Create("username", "test-username"), Tuple.Create("client-name", "test-client"), Tuple.Create("ssl", "true"));
+
+        // BindingType enabled
+        postProcessor.PostProcessConfiguration(GetConfigurationProvider(postProcessor, RedisPostProcessor.BindingTypeKey, true), configData);
+        Assert.Equal("test-host", configData["redis:client:host"]);
+        Assert.Equal("test-port", configData["redis:client:port"]);
+        Assert.Equal("true", configData["redis:client:ssl"]);
+        Assert.Equal("test-password", configData["redis:client:password"]);
+        Assert.Equal("test-client", configData["redis:client:name"]);
+        Assert.Throws<KeyNotFoundException>(() => configData["redis:client:username"]);
```

Comment:
```suggestion
```

Not copied or confused, just unnecessarily confirming this key isn't present, this was the key that we set to null in 4.0